tests/modules/services/git-sync/basic.nix

```diff
@@ -15,23 +15,13 @@
       serviceFile=home-files/.config/systemd/user/git-sync-test.service
 
       assertFileExists $serviceFile
-      assertFileContent $serviceFile ${
-        builtins.toFile "expected" ''
-          [Install]
-          WantedBy=default.target
-
-          [Service]
-          Environment=GIT_SYNC_DIRECTORY=/a/path
-          Environment=GIT_SYNC_COMMAND=@git-sync@/bin/git-sync
-          Environment=GIT_SYNC_REPOSITORY=git+ssh://user@example.com:/~user/path/to/repo.git
-          Environment=GIT_SYNC_INTERVAL=500
-          ExecStart=@git-sync@/bin/git-sync-on-inotify
-          Restart=on-abort
-
-          [Unit]
-          Description=Git Sync test
-        ''
-      }
+      assertFileContains $serviceFile "Environment=GIT_SYNC_DIRECTORY=/a/path";
+      assertFileContains $serviceFile "Environment=GIT_SYNC_COMMAND=@git-sync@/bin/git-sync"
+      assertFileContains $serviceFile "GIT_SYNC_REPOSITORY=git+ssh://user@example.com:/~user/path/to/repo.git"
+      assertFileContains $serviceFile "Environment=${
+        builtins.elemAt
+        config.systemd.user.services.git-sync-test.Service.Environment 0
+      }"
```

Comment:
Why this change? You can use `normalizeStorePaths` if you are concerned about the bin path changing.

Reply:
I am weirded out that each change to service needs to be 1-1 changed in the tests — it's just plain weird to make it false-positive any new addition as an error

Reply:
but ok

Reply:
That is the entire point of the tests. We want to ensure that every change to the output is intentional, not an accident.

Reply:
well, I fixed it